frontend/src/components/widgets/inputs/OptionalInput.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-	<LayoutRow class="optional-input">
+	<div class="optional-input">
 		<CheckboxInput :checked="checked" @input="(e) => $emit('update:checked', (e.target as HTMLInputElement).checked)" :icon="icon" />
-	</LayoutRow>
+	</div>
```

Comment:
Is this change necessary? It's done like this for uniformity with similar widgets and to generally avoid using `<div>`s for layout purposes (the general goal is to use divs for small styled elements rather than containers or things that are used for layout).

Reply:
Necessary to avoid this behaviour:
![image](https://user-images.githubusercontent.com/78500760/163704373-d8c35661-950c-4886-a56f-7dfb7b134a7c.png)

Reply:
It's better to just override the CSS property that is causing that problem.